### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Aladin Iseni, Arthur Jacobs

### DISCUSSION THREAD

**Discussion by aladin-heig** [heig-vd-dai-course/heig-vd-dai-course#528] [Show and tell]
Bonjour,

Pour le troisième travail pratique nous pensions réutiliser la base de notre second travail pratique par rapport au gestionnaire de notes. Notre client pourrait donc créer, modifier et supprimer des notes sur le serveur en utilisant curl et HTTP.

En option: faire une interface graphique au possible

**Comment by GeraudSilvestri** [comment]
Bon pour moi

**Comment by Arthur2479** [comment]
Here is the commit hash:
[ba17e46](https://github.com/HEIG-VD-DAI-Iseni-Jacobs/yes-tion/commit/ba17e46f4c6f9260cddc442a4375a329a7c73839)

And the link to our repo:
https://github.com/HEIG-VD-DAI-Iseni-Jacobs/yes-tion
